data/en/generatepbkdfkey.json

```diff
@@ -1,31 +1,37 @@
 {
 	"name":"generatePBKDFKey",
-	"syntax":"generatePBKDFKey(Algorithm, Passphrase, Salt, Iterations, KeySize)",
+	"syntax":"generatePBKDFKey(algorithm,passphrase,salt);",
```

Comment:
I saw that you removed `keySize` and `iterations` from the syntax... so I just tested to see if they were optional... They appear to be optional on Lucee but Required on ACF, we should probably keep them in the syntax. Also the param `required` is incorrect below, we should probably say they are required but note that they are optional on lucee.